tests/kafkatest/tests/streams/streams_eos_test.py

```diff
@@ -58,16 +58,16 @@ def run_rebalance(self, processor1, processor2, processor3, verifier):
 
         self.driver.start()
 
-        processor1.clean_node_enabled = False
-
         self.add_streams(processor1)
+        processor1.clean_node_enabled = False
```

Comment:
We can move the unsetting of the clean-up flag after the first start of `processor1` to ensure that the node is cleaned up during the start (see [code line in ducktape](https://github.com/confluentinc/ducktape/blob/f540a740222ca6eaf703d7315228e67b65db3aa9/ducktape/services/service.py#L228)).